feat: simplify creating agent using str, add examples (#1699)
camel-ai/camel: docs/key_modules/agents.md

response = agent.step("Hello, can you help me?")
```

### 3.2. Simplified Agent Creation
The `ChatAgent` supports multiple ways to specify the model:

```python
from camel.agents import ChatAgent
from camel.models import ModelFactory
from camel.types import ModelPlatformType, ModelType

> # Method 1: Using just a string for the model name (uses default platform)

Reviewer: ```suggestion
# Method 1: Using just a string for the model name (default model platform is used)
```

> agent1 = ChatAgent("You are a helpful assistant.", model="gpt-4o-mini")

Reviewer: Naming conventions should follow PEP 8, similarly for other agents below:
```suggestion
agent_1 = ChatAgent("You are a helpful assistant.", model="gpt-4o-mini")
```

> # Method 2: Using a ModelType enum

Reviewer: ```suggestion
# Method 2: Using a ModelType enum (default model platform is used)
```

agent2 = ChatAgent("You are a helpful assistant.", model=ModelType.GPT_4O_MINI)

# Method 3: Using a tuple of strings (platform, model)
agent3 = ChatAgent("You are a helpful assistant.", model=("openai", "gpt-4o-mini"))

# Method 4: Using a tuple of enums
agent4 = ChatAgent(
    "You are a helpful assistant.",
    model=(ModelPlatformType.ANTHROPIC, ModelType.CLAUDE_3_5_SONNET),
)

> # Method 5: Using default model when none is specified

Reviewer: ```suggestion
# Method 5: Using default model platform and default model type when none is specified
```
